src/query/expression/src/values.rs

```diff
@@ -1819,6 +1819,29 @@ impl Column {
         }
     }
 
+    pub fn serialize_size(&self) -> usize {
```

Comment:
why not use memory size directly?

Reply:
It is incorrect to use memory size directly, such as `Bitmap`, we will use more memory when serializing it.